### DISCUSSION TITLE
Did the light/dark theme switch disappear, or was it never there?

### DISCUSSION THREAD

**Discussion by spiralham** [louislam/dockge#281] [General]
I have two instances of Dockge 1.3.3 installed, somehow one of them is in light mode and the other is dark.  I swear I saw manual theme control in settings at one point, but now I don't.  Did I hallucinate the switch?  (for the record it's the same browser window in 2 different tabs, so if it's matching the theme to system they should both be the same)

![light](https://github.com/louislam/dockge/assets/7275942/4fc9d529-f697-47a6-bbc5-ef954df5f357)

![dark](https://github.com/louislam/dockge/assets/7275942/ebc6f9be-accf-4f23-882c-79a86c30e3f3)

**Comment by sottey** [comment]
Same. I feel like I am taking crazy pills.

**Comment by Node815** [comment]
Please add it back if it was missing, It makes it harder to see on my eyes.  While I do enjoy dark modes, sometimes I just need to step into the brighter side!  :sunglasses:    Don't get me wrong, it does look great dark but having a lighter color would be much nicer!

**Comment by nf9k** [comment]
Old and busted here, too.  All of mine are in dark-mode, but I swear I remember switching it on, but....?  Wrong week to quit sniffing glue?

**Comment by mguilherme** [comment]
Dark mode is harder for me, would really like to get light mode!

**Reply by cmcooper1980** [reply]
There is a Pull Request for this #575 that you can merge and build yourself if you really want to like I did in my fork. Once merged the toggle would be under Settings > Appearance and lets you toggle from (Light | Dark | Auto)
